frontend/src/js/components/settings/EnterpriseRequestExpanded.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import { useState } from 'react';
+
+import { Button, Drawer, FormControl, FormHelperText, TextField } from '@mui/material';
+
+import { DrawerTitle } from '@northern.tech/common-ui/DrawerTitle';
+import { AddonSelect } from '@northern.tech/common-ui/forms/AddonSelect';
+import { AddonIds } from '@northern.tech/store/constants';
+
+const note = 'Tell us a little about your requirements and device fleet size, so we can provide you with an accurate quote';
+
+interface EnterpriseRequestExpandedProps {
+  onClose: () => void;
+  onSendRequest: (e: { message: string; selectedAddons: AddonIds[] }) => void;
+  addons: { name: AddonIds }[];
+}
+
+export const EnterpriseRequestExpanded = (props: EnterpriseRequestExpandedProps) => {
+  const { onSendRequest, onClose, addons } = props;
+  const [message, setMessage] = useState('');
+  const [selectedAddons, setSelectedAddons] = useState<AddonIds[]>(addons.map(addon => addon.name));
+  return (
+    <Drawer anchor="right" open={true} PaperProps={{ style: { minWidth: '75vw' } }}>
+      <DrawerTitle onClose={onClose} title="Request for Mender Enterprise" />
+      <div className="flexbox column margin-bottom-large">
+        <div className="margin-top-large">
+          Get in touch with our team to request a quote for <span className="bold">Mender Enterprise.</span>{' '}
+        </div>
+        <FormControl style={{ marginBottom: 30, marginTop: 0, maxWidth: '550px' }}>
```

Comment:
```suggestion
        </div>
        <FormControl className="margin-bottom margin-top-none" style={{ maxWidth: 550 }}>
```
the margin classes _should_ work, otherwise please choose a way to reference the size (px or not)